src/core/thread/mle_router.cpp

```diff
@@ -2091,11 +2091,17 @@ otError MleRouter::UpdateChildAddresses(const Message &aMessage, uint16_t aOffse
         VerifyOrExit(aMessage.Read(offset, len, &entry) == len, error = OT_ERROR_PARSE);
 
         offset += len;
+        registeredCount++;
 
         if (entry.IsCompressed())
         {
-            // xxx check if context id exists
-            GetNetif().GetNetworkDataLeader().GetContext(entry.GetContextId(), context);
+            if (GetNetif().GetNetworkDataLeader().GetContext(entry.GetContextId(), context) != OT_ERROR_NONE)
+            {
+                otLogWarnMle(GetInstance(), "Error %s adding IPv6 address to child 0x%04x",
+                             otThreadErrorToString(error), aChild.GetRloc16());
```

Comment:
When we get the context `error` will not be set.  
How about saying something like "Failed to get context for compressed IPv6 address entry from child 0x%04x"?

Reply:
also output the context id?